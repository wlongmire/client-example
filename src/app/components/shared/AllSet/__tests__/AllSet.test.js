/* eslint-env node, jest */

import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import { AllSet } from '../index'

jest.mock('react-router', () => ({ browserHistory: [] }))

describe('>>> AllSet Component Snapshot', () => {
  it('+++capturing Snapshot with', () => {
    const renderedValue = shallow(
      <AllSet />)
    expect(toJson(renderedValue)).toMatchSnapshot()
  })
})