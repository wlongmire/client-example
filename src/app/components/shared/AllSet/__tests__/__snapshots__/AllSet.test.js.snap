// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>> AllSet Component Snapshot +++capturing Snapshot with 1`] = `
<div
  className="allSet"
>
  <i
    aria-hidden="true"
    className="fa fa-check-circle fa-3"
  />
  <p>
     View Your homepage to create and edit submissions. 
  </p>
  <Button
    active={false}
    block={false}
    bsClass="btn"
    bsStyle="default"
    disabled={false}
    onClick={[Function]}
  >
    View Hompeage
  </Button>
</div>
`;
